### PR TITLE
[200297] Antikythera.Time.new/1 should also be an identity mapping

### PR DESCRIPTION
**Pull request by irisTa56**
https://acsmine.tok.access-company.com/redmine/issues/200297#note-16

### CODE REVIEW

File: lib/util/time.ex [hunk 1]
```diff
@@ -37,8 +37,20 @@ defmodule Antikythera.Time do
...
+      {:ok, {#{__MODULE__}, {2015, 1, 23}, {23, 50, 7}, 0}}
+      iex> #{__MODULE__}.new(time)
+      {:ok, {#{__MODULE__}, {2015, 1, 23}, {23, 50, 7}, 0}}
+      iex> #{__MODULE__}.new("2015-01-23T23:50:07") |> Croma.Result.error?()
+      true
+      iex> #{__MODULE__}.new(nil) |> Croma.Result.error?()
+      true
   """
-  defun new(s :: v[IsoTimestamp.t()]) :: R.t(t), do: from_iso_timestamp(s)
+  defun new(t | IsoTimestamp.t()) :: R.t(t) do
```
Comment (aYukiSekiguchi) [inline lib/util/time.ex:50]: The document of this function says:
```
  Convert timestamps into `Antikythera.Time.t`, leveraging `recursive_new?` option of `Croma.Struct`.
```

but, `new/1` accepts `Antikythera.Time.t` now.

Could you fix the document?

**Review by aYukiSekiguchi — APPROVED** [review]
LGTM

File: lib/util/time.ex (hunk 1 above)
Comment (irisTa56) [reply lib/util/time.ex:51]: Thank you!
I should update the document.

fixed at https://github.com/access-company/antikythera/pull/204/commits/32fb7f59908af016a3badb59e9c4c6a892466136

**Review by aMasakiTakahashi — APPROVED** [review]
LGTM!

**Pull request merged**